packages/blade/src/components/Dropdown/dropdownUtils.ts

```diff
@@ -324,3 +325,58 @@ export const makeInputDisplayValue = (selectedIndices: number[], options: Option
   // When more than one item is selected, we display the count of items
   return `${selectedIndices.length} items selected`;
 };
+
+interface PositionProp {
+  top: number;
+  left: number;
+  right: number;
+  bottom: number;
+}
+
+export interface DropdownPosition {
+  top?: SpacingValueType;
+  bottom?: SpacingValueType;
+  left?: SpacingValueType;
+  right?: SpacingValueType;
+}
+
+export const POSITION_THRESHOLDS = {
+  top: -400,
+  bottom: -400,
+  left: -400,
+  right: -400,
```

Comment:
Where do these values come from?

Reply:
Since you have kept `max-width` for dropdown overlay as `400px`. I am reusing the same number to calculate height, width available in all positions

Reply:
Let's add a comment here.